scripts/test1.sh

```diff
@@ -0,0 +1,5 @@
+curl -X POST -H "Content-Type: application/json" --data @json/cvpc.json http://localhost:9009/project/3dda2801-d675-4688-a63f-dcda8d327f50/vpcs
+curl -X POST -H "Content-Type: application/json" --data @json/csn.json http://localhost:9009/project/3dda2801-d675-4688-a63f-dcda8d327f50/subnets
+curl -X POST -H "Content-Type: application/json" --data @json/node.json http://localhost:9007/v4/nodes
+curl -X POST -H "Content-Type: application/json" --data @json/sg.json http://localhost:9008/v4/3dda2801-d675-4688-a63f-dcda8d327f50/security-groups
```

Comment:
What is the expected result from current sg implementation?

Reply:
The expected result should be this:

{
   "security_group":{
      "id":"3dda2801-d675-4688-a63f-dcda8d111111",
      "project_id":"3dda2801-d675-4688-a63f-dcda8d327f50",
      "tenant_id":"3dda2801-d675-4688-a63f-dcda8d327f50",
      "name":"sg1",
      "description":"string",
      "security_group_rules":[
         {
            "id":"6cccca44-eac5-4b1f-9bbe-02157708916a",
            "project_id":null,
            "tenant_id":"3dda2801-d675-4688-a63f-dcda8d327f50",
            "name":null,
            "description":null,
            "security_group_id":"3dda2801-d675-4688-a63f-dcda8d111111",
            "remote_group_id":null,
            "direction":"egress",
            "remote_ip_prefix":null,
            "protocol":null,
            "port_range_max":null,
            "port_range_min":null,
            "ether_type":"IPv4"
         },
         {
            "id":"0ee2b64e-1405-4964-bd23-e48b50958126",
            "project_id":null,
            "tenant_id":"3dda2801-d675-4688-a63f-dcda8d327f50",
            "name":null,
            "description":null,
            "security_group_id":"3dda2801-d675-4688-a63f-dcda8d111111",
            "remote_group_id":null,
            "direction":"egress",
            "remote_ip_prefix":null,
            "protocol":null,
            "port_range_max":null,
            "port_range_min":null,
            "ether_type":"IPv6"
         }
      ],
      "create_at":"2020-07-25 00:06:28",
      "update_at":"2020-07-25 00:06:28"
   }
}